doc/documentation.rst

```diff
@@ -23,16 +23,13 @@ See the examples section for full code snippets.
 .. include:: includes/big_toc_css.rst
 
 .. toctree::
-   :maxdepth: 2
-
-   userguide_tablereport
-   userguide_encoders
-   userguide_datetimes
-   userguide_tablevectorizer
-   userguide_data_cleaning
-   userguide_selectors
-   userguide_data_ops
-   userguide_data_ops_ml_pipeline
-   userguide_data_ops_validation
-   userguide_joining_tables
-   userguide_utils
+   :maxdepth: 3
+
+   exploring_a_dataframe
+   cleaning_a_dataframe
+   encoding_a_column
+   building_a_pipeline
+   selectors
```

Comment:
the first 4 titles are great, but "selectors" and "dataops" are much less informative. could we change them to match the styles of the previous ones, describing the task rather than naming the module? I know it's hard to do in a short title

Reply:
or maybe each section of the TOC could include a short sentence or paragraph below the title to say what this is about and who might be interested to read it